packages/wow-ui/src/components/Box/index.tsx

```diff
@@ -1,25 +1,130 @@
-import { css } from "@styled-system/css/css";
-import type { ReactNode } from "react";
+import { cva } from "@styled-system/css";
+import { Flex, styled } from "@styled-system/jsx";
+import type { CSSProperties, ReactNode } from "react";
+import { RightArrow } from "wowds-icons";
+import { color } from "wowds-tokens";
 
-export interface ButtonProps {
-  children: ReactNode;
+import Checkbox from "@/components/Checkbox";
+
+export interface BoxProps {
+  leftElement?: ReactNode;
+  type?: "text" | "checkbox" | "arrow";
+  text: string;
+  textColor?: string;
+  subText?: string;
+  subTextColor?: string;
+  status?: "default" | "success" | "error";
+  onClick?: () => void;
+  style?: CSSProperties;
 }
 
-const Box = ({ children }: ButtonProps) => {
+/**
+ * @description 사용자에게 보여주어야 하는 정보를 담을 수 있는 Box 컴포넌트입니다.
+ * @param {ReactNode} [leftElement] Box 컴포넌트의 왼쪽에 들어갈 수 있는 요소입니다. (아이콘, 이미지 등)
+ * @param {string} [textColor] text의 색상을 변경할 수 있습니다.
+ * @param {"text" | "checkbox" | "arrow"} [type] Box 컴포넌트의 타입을 설정합니다.
+ * @param {string} text Box 컴포넌트에 메인으로 표기할 텍스트를 입력합니다.
+ * @param {string} [subText] Box 컴포넌트에 작성할 추가 정보를 입력합니다.
+ * @param {string} [subTextColor] subtext의 색상을 변경할 수 있습니다.
+ * @param {"default" | "success" | "error"} [status] Box 컴포넌트를 통해 사용자의 상태를 반환합니다.
+ * @param {() => void} [onClick] Box 컴포넌트의 타입이 "checkbox"와 "arrow"일때 수행할 onClick 함수를 입력합니다.
+ * @throws {onClick} onClick 함수는 "text" type에서 사용할 수 없습니다.
+ * @param {CSSProperties} [style] Box 컴포넌트에 적용할 수 있는 custom style
+ */
+
+const Box = ({
+  leftElement,
+  type = "text",
+  text,
+  textColor,
+  subText,
+  subTextColor,
+  status = "default",
+  onClick,
+  style,
+  ...rest
+}: BoxProps) => {
   return (
-    <button
-      className={css({
-        bg: "blue.100",
-        fontFamily: "Inter",
-        px: "4",
-        py: "3",
-        borderRadius: "md",
-        _hover: { bg: "blue.400" },
-      })}
+    <Flex
+      alignItems="center"
+      className={containerStyle({ status: status })}
+      direction="row"
+      gap="lg"
+      id={`box-${text}`}
+      justifyContent="space-between"
+      style={{ ...style }}
+      {...rest}
     >
-      {children}
-    </button>
+      <Flex alignItems="center" direction="row" gap="xs">
+        {leftElement}
+        <Flex direction="column" gap="xxs">
+          <styled.span
+            color="red"
+            fontWeight="600"
+            textStyle="h3"
+            style={{
+              color: textColor ? textColor : `${color.textBlack}`,
+            }}
```

Comment:
p4;
위에서 지정한 color='red' 는 style prop 으로 전달한 color 프로퍼티가 우선 적용되어 red 로 나오는 경우가 없을 것 같아요..!! style prop 으로 전달한 프로퍼티를 color 프로퍼티로 전달해도 될 것 같숩니당
```suggestion
            color={textColor ? textColor : "textBlack"}
```

Reply:
아무래도 panda css가 정적인 CSS툴이다 보니까 예시로 들어주신 것과 같이[ 동적으로 스타일이 적용](https://panda-css.com/docs/guides/dynamic-styling#jsx-style-props)되지 않는것같아요 ㅠ 아무래도 기존에 사용하던 `style`객체를 활용하는 쪽으로 생각해야 겠습니다아.. 
```ts
import { useState } from 'react'
 
const App = () => {
  const [color, setColor] = useState('blue.300')
 
  // ❌ Avoid: Panda can't determine the value of color at build-time
  return (
    <Card color={color}>
      <p>Some content</p>
    </Card>
  )
}
```

Reply:
결론 ) panda config 파일 수정해서 static css설정해두기로 했어요!